src/style/globalStyles.ts

```diff
@@ -66,7 +66,7 @@ export const GlobalStyle = createGlobalStyle`
     --fontWeight-semiBold: 600;
     --fontWeight-bold: 800;
 
-    --inputHeight: 50px;
+    --inputHeight: 60px;
```

Comment:
I increased it because inputs that display badges needed more space

Reply:
`60px` is a bit much IMO. Let's bump it to `55px` by default, and use a `isDisplayingBadges` props on the input if necessary?

Reply:
Or we add a height prop, with predefined values like `short | medium | tall`.
When used in list, a shorter version (45px) looks better than:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/6715943/220176477-832ec3f7-539e-456e-95d5-22f43b4a1e49.png">